src/general/OptionsMenu.cs

```diff
@@ -2384,4 +2404,12 @@ private void OnOpenPatchNotesPressed()
         patchNotesDisplayer.ShowLatest();
         patchNotesBox.PopupCenteredShrink();
     }
+
+    private void OnScreenEffectSelected(int index)
```

Comment:
These methods should be in the same order as the node variables (so this should be next to the one about JSON debug mode).